data/error_list.json

```diff
@@ -18,7 +18,7 @@
 	  "pattern": [
 		{
 		  "id": "C_RELOAD_CHAR_MODELS"
-		}
+		},
```

Comment:
You don't need to do a comma here if there isn't another control after this one.